requirements/dev.txt

```diff
@@ -13,10 +13,12 @@ black==20.8b1             # via -r requirements/dev.in
 certifi==2020.4.5.1       # via -r requirements/ci.txt, elastic-apm, requests, sentry-sdk
 chardet==3.0.4            # via -r requirements/ci.txt, requests
 click==7.1.2              # via black, pip-tools
+commonmark==0.9.1         # via recommonmark
 coreapi==2.3.3            # via -r requirements/ci.txt, drf-yasg
 coreschema==0.0.4         # via -r requirements/ci.txt, coreapi, drf-yasg
 coverage==4.5.4           # via -r requirements/ci.txt
 cssselect==1.1.0          # via -r requirements/ci.txt, pyquery
+dataclasses==0.7          # via black
```

Comment:
Do you use python 3.7 or above? They already include `dataclasses` in standard lib for python >= 3.7, therefore `pip install -r requirements/dev.txt` fails with error now

Reply:
Woops, I used Python 3.6